<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="image_view" rowminimumheight="0" columnminimumwidth="0">
    <property name="spacing">
     <number>5</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="submit_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="select_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Image Count: 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_field">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Selection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
